test/test_cuda_mapper_memory_promotion.cc

```diff
@@ -260,9 +260,9 @@ def fun(float(N, M) A, float(N, M) B) -> (C) {
     EXPECT_EQ(groups.size(), 3u);
 
     USING_MAPPING_SHORT_NAMES(BX, BY, BZ, TX, TY, TZ);
-    isl::space blockSpace = isl::space(ctx, 0);
-    isl::set blockZero =
-        isl::makeSpecializationSet<int>(blockSpace, {{BX, 0}, {BY, 0}});
+    isl::space tileSpace = isl::space(ctx, 0).unnamed_set_from_params(2);
+    auto tileZero =
+        isl::map(isl::multi_aff::zero(tileSpace.from_range())).range();
```

Comment:
This looks like a very intricate way of constructing a 2-d set where both dimensions are equal to 0.  A comment would be appreciated.

I would also prefer the syntax we use for individual affs (something along the lines of `aff_set(isl::multi_aff::identity(tileSpace)) == isl::multi_val::zero(tileSpace)`), but we don't have those operator overloads for `multi_*` yet.  Not for this PR.